### DISCUSSION TITLE
Unified inbox?

### DISCUSSION THREAD

**Discussion by mreed4** [bkaankose/Wino-Mail#138] [Ideas]
So far this looks like a fine application. Great job!

In my quest for a replacement for the trash that all iterations of Outlook have become, I came across [Spark](https://sparkmailapp.com/features). I used it for a while, but its interface I just didn't really click with. Basically, it is a super minimal interface, with a unified inbox. I think it is designed for OS X, but there is an exe for Windows as well. 

I digress.

Do you have a plan to make a unified inbox as an option? Understand if it is not part of your plans!

Thank you for building this app!

**Comment by kbuckleys** [comment]
I've been using Missive as my daily driver, and I do miss the unified inbox already. Hope it's in the pipes.

**Comment by bkaankose** [comment]
Forgot to mention, but unified inbox feature is already available for beta users on v1.7.0+

You can give it a try here:

https://github.com/bkaankose/Wino-Mail#beta-releases

**Reply by kbuckleys** [reply]
This is great. I'll stick to the stable release and hopefully an update will be pushed soon. Thank you.
